.vscode/settings.json

```diff
@@ -21,5 +21,8 @@
     "node_modules/**": true,
     "packages/node_modules/**": false
   },
-  "eslint.autoFixOnSave": true
+  "eslint.autoFixOnSave": true,
```

Comment:
Is this change required for the node upgrade?

Reply:
Nope, vscode did that on it's own
Although I'm not opposed to adding it in as its own commit in the PR since it will probably show up in everyone elses PRs

Reply:
Sounds good, let's put it in a commit.